activerecord/lib/active_record/encryption/configurable.rb

```diff
@@ -9,6 +9,7 @@ module Configurable
       included do
         mattr_reader :config, default: Config.new
         mattr_accessor :encrypted_attribute_declaration_listeners
+        mattr_reader :encrypted_attributes, default: Concurrent::Array.new
```

Comment:
I like the hash version better you refer at https://github.com/rails/rails/pull/48520/commits/95b3e01cdce2b38af8c721f23dab4858b766ce3e. 

Also, I'd consider renaming `:encrypted_attributes` to `entrypted_attributes_per_class` to clarify a little bit what's contained there.